command/kv_put.go

```diff
@@ -219,6 +219,11 @@ func (c *KVPutCommand) Run(args []string) int {
 		return PrintRawField(c.UI, secret, c.flagField)
 	}
 
+	// If the secret is wrapped, return the wrapped response.
+	if secret != nil && secret.WrapInfo != nil && secret.WrapInfo.TTL != 0 {
```

Comment:
Same comment here as above - we already check `if secret == nil` on line 210, so no need to check again here.

Reply:
```suggestion
	if secret.WrapInfo != nil && secret.WrapInfo.TTL != 0 {
```

Reply:
Thanks!